src/cljs/nr/cardbrowser.cljs

```diff
@@ -213,7 +182,7 @@
    [:div.text
     [:p [:span.type (str (:type card))] (if (empty? (:subtype card))
                                           "" (str ": " (:subtype card)))]
-    [:pre {:dangerouslySetInnerHTML #js {:__html (add-symbols (:text card))}}]
+    [:pre (render-icons (:text (first (filter #(= (:title %) (:title card)) @all-cards))))]
```

Comment:
```suggestion
    [:pre (render-icons (:text (get @all-cards (:title card))))]
```
Does the original work? `@all-cards` should be a map with the card title as key. You could also do `(get-in @all-cards [(:title card) :text])` if you prefer the look of that.

Reply:
Nope the original always renders and empty pre-tag (`add-symbols` was permissive and would spit out empty string if given `nil`)

Sadly `@all-cards` is a sequence of maps:
```=> (take 2 @all-cards)
({:faction "Neutral",
  ...
  :title "The Shadow: Pulling the Strings",
  ...
  :text
  "Draft format only.\nYou can use agendas from all factions in this deck."}
 {:faction "Neutral",
  ...
  :title "The Masque: Cyber General",
  ...
  :text "Draft format only."})
```
and I couldn't figure out a more elegant way to query that structure.

Reply:
Yikes. Why is that happening? Let me look into this, cuz that shouldn't be the way it is...